src/Bowling.js

```diff
@@ -0,0 +1,33 @@
+function Bowling() {
+  this.total = 0;
+  this.totalFrames = 10;
+  this.frame = [];
+  this.strike = 10;
+  this.spare =10;
+};
+
+Bowling.prototype.rolls = function(roll_1, roll_2, frame){
+    this.frame[frame]=[roll_1,roll_2];
```

Comment:
Identifier 'roll_1' is not in camel case.<br>Identifier 'roll_2' is not in camel case.